openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -64,13 +76,18 @@ def process(self, instance):
 
         # Perform extraction
         self.log.debug("Performing extraction ...")
+        attribute_values = self.get_attr_values_from_data(
+            instance.data
+        )
         with maintained_selection():
             cmds.select(selection, noExpand=True)
             cmds.file(path,
                       force=True,
                       typ="mayaAscii" if self.scene_type == "ma" else "mayaBinary",  # noqa: E501
                       exportSelected=True,
-                      preserveReferences=True,
+                      preserveReferences=attribute_values[
+                          "preserve_references"
+                      ],
```

Comment:
I believe there _may_ be cases where the attribute defs are not defined. I see these values often retrieved with `.get()`. Maybe @iLLiCiTiT knows more why that is and whether it should.
```suggestion
                      preserveReferences=attribute_values.get(
                          "preserve_references", True
                      ),
```

Reply:
To be honest I can't imagine case when it would not be there, only running standard pyblish.api logic could cause it's not there, but in that case would probably be missing all instances...

Reply:
@BigRoy you still want it in there? Seems like if its not there, we would have bigger issues.

Reply:
No fine with me - if someone can do a test run of course.